src/useEventListener.js

```diff
@@ -2,7 +2,7 @@
 
 import { useRef, useEffect } from 'react';
 
-function useEventListener(eventName, handler, element = window) {
+function useEventListener(eventName, handler, element = typeof window !== "undefined" ? window : undefined) {
```

Comment:
There's a utility in `/utils` called `isClient`. We should use that instead. Also, make sure to update the tests so the build passes. This is a good idea for a PR, thank you